pkg/model/components/docker.go

```diff
@@ -54,7 +54,7 @@ func (b *DockerOptionsBuilder) BuildOptions(o interface{}) error {
 
 		dockerVersion := ""
 		if sv.Major == 1 && sv.Minor >= 17 {
-			dockerVersion = "19.03.4"
+			dockerVersion = "19.03.11"
```

Comment:
This should be changed in master branch first, so they don't downgrade Docker when they upgrade kops.

Reply:
Will create PR for that also, but didn't want another cherry-pick for this thing also...

Reply:
https://github.com/kubernetes/kops/pull/9317

Reply:
I'm holding LGTM until #9317 lands.